.github/workflows/code-quality.yml

```diff
@@ -16,9 +16,15 @@ jobs:
         uses: actions/cache@v4
         with:
           path: ~/.npm
-          key: ${{ runner.os }}-npm-${{ hashFiles('**/workflows/prettier.yml') }}
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/workflows/code-quality.yml') }}
           restore-keys: |
             ${{ runner.os }}-npm-
 
+      - name: Install dependencies
+        run: npm ci
```

Comment:
We should probably update readme as well.

Reply:
Good point, added now ✅